### ISSUE TITLE
ADD TO QUEUE WONT WORK 

### ISSUE THREAD

**Issue by BoMshmsh** [mrpond/BlockTheSpot#590]
ADD TO QUEUE WONT WORK , It worked previously

**Comment by toontownlpz** [comment]
Can confirm I am also having this same issue. Adding a song to the queue 'works' per se but Spotify will just freeze up when it reaches the end of the currently playing song, and will not move onto the next song in the queue.

**Comment by strawberryhai-j** [comment]
yeah no.. songs don't auto play once they end.. and it stops the queue or playlist if you try to skip songs as well.. Spotify likely did something to combat ad blockers.. or something in the code broke.. I'm hoping it gets patched soon.. I cant stand these ads anymore...

**Comment by SiddheshNan** [comment]
Having the same issue... next song is not playing from any playlist or queue.

**Comment by Lufthansaaa** [comment]
Same issue, no song is plays after the first song.

**Comment by Bramhutls** [comment]
> Not related to this repo, It's a spotify issue.
> 
> https://community.spotify.com/t5/Ongoing-Issues/Playback-issues-related-to-the-next-song-in-the-queue/idi-p/6606114
> 
> **Solution:** Just uninstall and reinstall spotify.

Reinstalling did not solve the issue for me, queue works without "BlockTheSpot with Spicetify" installed.

**Comment by imperdiet** [comment]
I'm glad I wasn't the only one having this issues

**Comment by Lufthansaaa** [comment]
Anybody found a fix yet?

**Comment by chorbintime** [comment]
Randomly fixed for me today with a Spotify update

**Comment by Lufthansaaa** [comment]
Fixed for me too, just re-run your script.

**Comment by mrpond** [comment]
It's Spotify issue. update Spotify should solve the problem.
